en/dev_log/ulog_file_format.md

```diff
@@ -29,12 +29,15 @@ The following binary types are used for logging. They all correspond to the type
 |double             |  8            |
 |bool, char         |  1            |
 
-Additionally the types can be used as an array : eg. `float[5]`. In case of logging of strings : (`char[length]`), it usually does not contain the termination NULL character `'\0'` at the end.
+Additionally the types can be used as an array: e.g. `float[5]`.
+
+Strings (`char[length]`) do not usually contain the termination NULL character `'\0'` at the end.
 String comparisons are case sensitive.
```

Comment:
@bkueng I am still wondering if this part is still relevant / necessary for the User Doc. Could we bring a contextual example that shows when 'case sensitie-ness' matters?

Is it used in Flight Review? ulog python library? etc.

Reply:
It's relevant, don't just remove things. This doc is meant to be a semi-formal file format spec, that can be used to implement a file reader or writer. Please keep that in mind.